content/applications/inventory_and_mrp/repairs/repair_orders.rst

```diff
@@ -0,0 +1,269 @@
+   :align: center
+   :alt: Right-hand side of blank repair order form.
+
+In the :guilabel:`Responsible` field, click the drop-down menu and select the user who should be
+responsible for the repair.
+
+In the :guilabel:`Company` field, if in a multi-company environment, select which company this |RO|
+belongs to.
+
+In the :guilabel:`Tags` field, click the drop-down menu and select which tags should be applied to
+this |RO|.
+
+Parts tab
+---------
+
+Add, remove, or recycle parts in the :guilabel:`Parts` tab. To do so, click :guilabel:`Add a line`
+at the bottom of the form.
+
+In the :guilabel:`Type` column, click the box to reveal three options to choose from:
+:guilabel:`Add` (selected by default), :guilabel:`Remove`, and :guilabel:`Recycle`.
+
+.. image:: repair_orders/repair-orders-type-column.png
+   :align: center
+   :alt: Type column options or new part under Parts tab.
+
+Choosing :guilabel:`Add` adds this part to the |RO|. Adding parts lists components for use in the
+repair. If the components are used, the user completing the repair can record they were used. If
+they were not used, the user can indicate that, too, and the components can be saved for another
+use.
+
+Choosing :guilabel:`Remove` removes this part from the |RO|. Removing parts lists components that
+should be removed from the product being repaired during the repair process. If the parts are
+removed, the user completing the repair can indicate they were removed.
+
+Choosing :guilabel:`Recycle` recycles this part from the |RO|, designating it for later use or to be
+repurposed for another use in the warehouse.
+
+In the :guilabel:`Product` column, select which product (part) should be added, removed, or
+recycled. In the :guilabel:`Demand` column, change the quantity, if necessary, to indicate what
+quantity of this part should be used in the repair process.
+
+In the :guilabel:`Done` column, change the value (in a `0.00` format) once the part has been
+successfully added, removed, or recycled.
+
+In the :guilabel:`Unit of Measure` column, select the |UoM| for the part.
+
+Finally, in the :guilabel:`Used` column, tick the checkbox once the part has been used in the repair
+process.
+
+To add additional columns to the line, click the :guilabel:`(slider)` icon, at the far-right of the
+header row. Select the desired options to add to the line.
+
+.. image:: repair_orders/repair-orders-additional-options.png
+   :align: center
+   :alt: Optional additional options to add to new part line.
+
+Repair Notes and Miscellaneous tabs
+-----------------------------------
+
+Click the :guilabel:`Repair Notes` tab to add internal notes about this specific |RO|, and anything
+the user performing the repair might need to know.
+
+Click the blank text field to begin writing notes.
+
+Click the :guilabel:`Miscellaneous` tab to see the :guilabel:`Operation Type` for this repair. By
+default, this is set to :guilabel:`YourCompany: Repairs`, indicating this is a repair type
```

Comment:
Just double checking- should there be a space here?
```suggestion
default, this is set to :guilabel:`Your Company: Repairs`, indicating this is a repair type
```

Reply:
There's not a space, although I know it looks weird :-/ But that's how it appears in the UI